data/civ.yaml

```diff
@@ -0,0 +1,6 @@
+name: Côte d'Ivoire
+official_name: 	The Republic of Côte d'Ivoire
```

Comment:
Can you remove the extra spaces, please?
```suggestion
official_name: The Republic of Côte d'Ivoire
```

Reply:
Am on it